apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -18,10 +27,78 @@ FluentUI supports browsers that are at or above the indicated versions below:
 | ------------- | ------------------ | ------- |
 | >=14.5        | >=84               | >=16    |
 
-FluentUI will ensure that no browser, EcmaScript(ES), or CSS features that are incompatible with the support matrix laid out above are introduced.
-FluentUI will not be providing polyfills for features that we expect our browser support matrix to already support. This means that consumers who require the
-use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. `ResizeObserver`),
-are expected to implement their own polyfills or use a third-party one.
+FluentUI will ensure that no browser, ECMAScript, or CSS features that are incompatible with the support matrix laid out above are introduced.
+For consumers who require the use/support of older browsers, please see the partial support matrix and polyfill sections below for more information
+on what will need to be done.
```

Comment:
[just a comment] This paragraph is a great improvement, see how it allows the reader to finish at this point. That is useful for the reader to solidity what we've just described as an independent concept. This serves kind of a bookmark in their understanding of the topic, now we are going to build on and elaborate on this core concept further.